code/parse/sexp.cpp

```diff
@@ -11968,7 +11964,7 @@ void sexp_exchange_cargo(int n)
 	// we must be sure that these two objects are indeed docked
 	if (!dock_check_find_direct_docked_object(&Objects[Ships[shipnum1].objnum], &Objects[Ships[shipnum2].objnum]))
 	{
-		Int3();			// you are trying to transfer cargo between two ships not docked
+		Error(LOCATION, "Tried to transfer cargo between %s and %s although they aren't docked!", Ships[shipnum1].ship_name, Ships[shipnum2].ship_name);
```

Comment:
Here too. Also, the message should say "exchange" rather than "transfer". (This sexp swaps the cargo; the other sexp moves it from one to the other, possibly overwriting the cargo in the destination.)